packages/muelu/research/mmayr/composite_to_regions/src/SetupRegionHierarchy_def.hpp

```diff
@@ -92,18 +93,18 @@ using Teuchos::ParameterList;
  * This can be achieved by setting \c inverseScaling to \c true.
  */
 template <class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
-void scaleInterfaceDOFs(std::vector<RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> > >& regVec, ///< Vector to be scaled
-    const std::vector<RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> > >& scalingFactors, ///< Vector with scaling factors
-    bool inverseScaling ///< Divide by scaling factors (yes/no?)
-    )
+void scaleInterfaceDOFs(Array<RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> > >& regVec, ///< Vector to be scaled
```

Comment:
Regarding the above "def" includes -- why do you need those?  Are you using types for which MueLu does not do ETI?

Reply:
No but this code is not meant to survive all that long so little effort is made to clean it.
A complete redesign will happen before we attempt to move anything out of `Experimental`.
For instance most of the linear algebra related functions will eventually be pulled into classes.